### DISCUSSION TITLE
like a last read symbol

### DISCUSSION THREAD

**Discussion by swschilke** [dailydotdev/daily#1319] [Feature Requests]
a symbol which shows me til where I have read the cards with news on my last visit, I want a way / mode to see only new news cards.

**Comment by idoshamun** [comment]
Hi @swschilke,

Thanks for the suggestion! Our feed ranking is not consistent, meaning the order of cards may change.
Hence, we can't support this feature yet. We're considering changing this behavior, but it has not been prioritized yet.

**Comment by swschilke** [comment]
Thank you - what about you add an extra feed (call it Timeline on the left
menue) and I can set this as my default. The feed could be your ranked
cards view as it is. Noboddy get's hurt and everybody is happy ;-)

**Reply by idoshamun** [reply]
We have a feed that is sorted by time if you want to use it but we don't have the mark you're asking for

**Comment by swschilke** [comment]
Thank you is that my Feed or where can I find that?

**Reply by idoshamun** [reply]
Head over to customize section by clicking on your profile picture
<img width="265" alt="image" src="https://github.com/user-attachments/assets/6b9a731e-7568-43f1-b832-c441ad6a483a">

Then you should see an option to enable feed sorting
<img width="242" alt="image" src="https://github.com/user-attachments/assets/ce317558-8de8-4f17-bbcc-d967e0d3ee74">

**Comment by swschilke** [comment]
I happily report that this works with the up/down arrows

**Reply by idoshamun** [reply]
Perfect!
